azure/scope/machinepoolmachine.go

```diff
@@ -396,160 +412,6 @@ func (s *MachinePoolMachineScope) UpdateInstanceStatus(ctx context.Context) erro
 	return nil
 }
 
-// CordonAndDrain will cordon and drain the Kubernetes node associated with this AzureMachinePoolMachine.
```

Comment:
This gets deleted because we're reusing the Machine implementation of `CordonAndDrain`? 👏🏻

Reply:
Yep!